doc/tools/apigen.py

```diff
@@ -268,6 +268,9 @@ def generate_api_doc(self, uri):
         if not (len(functions) or len(classes) or len(submodules)) and DEBUG:
             print('WARNING: Empty -', uri)
             return ''
+        functions = sorted(functions)
```

Comment:
Maybe we should make this behavior toggle-able. I would add a flag as a class attribute to `ApiDocWriter`. What do others think?

Reply:
Seems complicated. Either we want things sorted or not. Let's leave it at that.

Reply:
Works for me. :+1: